Client/Application/AppDelegate.swift

```diff
@@ -170,10 +170,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
 
         let fxaLoginHelper = FxALoginHelper.sharedInstance
         fxaLoginHelper.application(application, didLoadProfile: profile)
-        
-        if let url = launchOptions?[UIApplicationLaunchOptionsKey.url] as? URL, let path = NavigationPath(url: url) {
```

Comment:
I thought we needed this for when the app is launching via a deep link. But turns out you dont. the system will call `application(_ application: UIApplication, open url: URL` even if the app is launching.